src/icon/icon.jsx

```diff
@@ -40,14 +46,15 @@ const Icon = ({
   )
 
   // Label component
-  const label = labelFactory(iconCode)
+  const label = labelFactory(iconCode, alt)
 
   // Final component
   return (
     <div style={style} className={classes}>
-      {labelLeft && label(labelLeft)}
+      {!alt && labelLeft && label(labelLeft)}
       {iconComponent}
-      {labelRight && label(labelRight)}
+      {!alt && labelRight && label(labelRight)}
+      {alt && <span className={styles.srOnly}>{alt}</span>}
```

Comment:
this element is in the DOM, but visually hidden, it'll be displayed only if CSS is disabled